hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -117,21 +118,33 @@ def submit_content_request_from_s3_event_record(
 ) -> SubmitContentRequestBody:
     """
     Converts s3 event into a SubmitContentRequestBody object with a URL to the content
+
+    For partner bucket uploads the content IDs are unique and human understandable but
```

Comment:
nit: human understandable seems like a weird phrase here. Depends on completely on file name conventions, maybe just "Part of the original file name is include in ID to help with possible debugging but do not depend on this value, actual reversible look up should refer to the objects additional fields for a URL corresponding to the bucket and object name."